compute/metadata/retry.go

```diff
@@ -43,6 +43,9 @@ type metadataRetryer struct {
 }
 
 func (r *metadataRetryer) Retry(status int, err error) (time.Duration, bool) {
+	if status == 200 {
```

Comment:
Nit: should we use `http.StatusOK`?

Reply:
Good idea, also added a test that calls the wrapping code for good measure.